docs/source/tokenizer_summary.rst

```diff
@@ -1,223 +1,243 @@
 
-Subword tokenization algorithms rely on the principle that most common words should be left as is, but rare words
-should be decomposed in meaningful subword units. For instance "annoyingly" might be considered a rare word and
-decomposed as "annoying" and "ly". This is especially useful in agglutinative languages such as Turkish, where you can
-form (almost) arbitrarily long complex words by stringing together some subwords.
+Subword tokenization algorithms rely on the principle that frequently used words should not be split into smaller
+subwords, but rare words should be decomposed into meaningful subwords. For instance "annoyingly" might be considered a
+rare word and could be decomposed into "annoying" and "ly". Both "annoying" and "ly" as stand-alone sub-words would
+appear more frequently while at the same time the meaning of "annoyingly" is kept by the composite meaning of
+"annoying" and "ly". This is especially useful in agglutinative languages such as Turkish, where you can form (almost)
+arbitrarily long complex words by stringing together subwords.
 
-This allows the model to keep a reasonable vocabulary while still learning useful representations for common words or
-subwords. This also enables the model to process words it has never seen before, by decomposing them into subwords it
-knows. For instance, the base :class:`~transformers.BertTokenizer` will tokenize "I have a new GPU!" like this:
+Subword tokenization allows the model to have a reasonable vocabulary size while being able to learn meaningful
+context-independent representations. In addition, subword tokenization enables the model to process words it has never
+seen before, by decomposing them into known subwords. For instance, the :class:`~transformers.BertTokenizer` tokenizes
+"I have a new GPU!" as follows:
 
 .. code-block::
 
     >>> from transformers import BertTokenizer
-    >>> tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
+    >>> tokenizer = BertTokenizer.from_pretrained("bert-base-uncased")
     >>> tokenizer.tokenize("I have a new GPU!")
-    ['i', 'have', 'a', 'new', 'gp', '##u', '!']
+    ["i", "have", "a", "new", "gp", "##u", "!"]
 
-Since we are considering the uncased model, the sentence was lowercased first. Then all the words were present in the
-vocabulary of the tokenizer, except for "gpu", so the tokenizer splits it in subwords it knows: "gp" and "##u". The
-"##" means that the rest of the token should be attached to the previous one, without space (for when we need to decode
-predictions and reverse the tokenization).
+Because we are considering the uncased model, the sentence was lowercased first. We can see that the words ``["i",
+"have", "a", "new"]`` are present in the tokenizer"s vocabulary, but the word "gpu" is not. Consequently, the tokenizer
+splits "gpu" into known subwords: ``["gp" and "##u"]``. "##" means that the rest of the token should be attached to the
```

Comment:
```suggestion
splits "gpu" into known subwords: ``["gp" and "##u"]``. ``"##"`` means that the rest of the token should be attached to the
```

Reply:
yes - applied to all standalone "<word>" occurances.